utils/bots/TutorBot/cogs/TutorStaff.py

```diff
@@ -75,97 +82,32 @@ async def remove(self, ctx, id):
                 value=f"**Session ID:** `{query.SessionID}`"
                 f"\n**Student ID:** `{query.StudentID}`",
             )
-            await ctx.send(embed=embed)
+            await ctx.respond(embed=embed)
 
         else:
             embed = discord.Embed(
                 title="Invalid Session",
                 description="This session does not exist, please check the ID you've provided!",
                 color=discord.Color.red(),
             )
-            await ctx.send(embed=embed)
-
-    @commands.command()
-    @commands.has_role("Tutor")
-    async def schedule(
```

Comment:
If you're going to delete this, perhaps check the schedule slash command in main.py (~line 209) and update the guild id and permissions there?

Reply:
why was this command limited to tutors but the other wasnt?

Reply:
🤷, I'm pretty sure it was limited to the tutor server so we just assumed everyone on it was staff.

Reply:
oh i see